tests/tests_deployment/test_jupyterhub_api.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from tests.tests_deployment import constants
+from tests.tests_deployment.utils import get_jupyterhub_session
+
+
+@pytest.mark.filterwarnings("ignore::urllib3.exceptions.InsecureRequestWarning")
+def test_jupyterhub_loads_roles_from_keycloak():
+    session = get_jupyterhub_session()
+    xsrf_token = session.cookies.get("_xsrf")
+    response = session.get(
+        f"https://{constants.NEBARI_HOSTNAME}/hub/api/users/{constants.KEYCLOAK_USERNAME}",
+        headers={"X-XSRFToken": xsrf_token},
+        verify=False,
+    )
+    user = response.json()
+    assert set(user["roles"]) == {
```

Comment:
This is passing locally but failing on CI. I suspect this is because the CI is using docker images with older JupyterHub versions - if I am right, merging https://github.com/nebari-dev/nebari-docker-images/pull/140 should make the tests green.